examples/analog_out/cont_gen_voltage_wfm_int_clk_non_regen.py

```diff
@@ -0,0 +1,54 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a continuous periodic
+waveform with new data using an internal sample clock while
+the task is running.
+"""
+
+import time
+
+import numpy as np
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, RegenerationMode
+
+
+def get_sine_wave_data(cycle):
+    """Generate a sine wave data."""
+    frequency = (10, 50)
+    amplitude = (1, 3)
+    sampling_rate = (100, 500)
+    duration = 1
+    selection = cycle % 2
+
+    # generate the time array
+    t = np.arange(0, duration, 1 / sampling_rate[selection])
+    # Generate the sine wave
+    data = amplitude[selection] * np.sin(2 * np.pi * frequency[selection] * t)
+    return data
```

Comment:
This does not preserve phase from call to call, so it will generate a glitch in the sine wave for some frequencies.

Reply:
Yeah, it's removed. The example will demo how to generate a freq signal is not even divide-down by sample clock as per suggestion.